spec/factories/collections.rb

```diff
@@ -40,4 +41,12 @@
 
     depositors { build_list(:user, depositor_count) }
   end
+
+  trait :depositing_state do
+    state { 'depositing' }
+  end
+
+  trait :first_draft_state do
+    state { 'first_draft' }
+  end
```

Comment:
Not a big deal, but the Work factory defines related traits without the `_state` suffix on them. They don't need to evolve in lockstep, of course.